pages/osx/dockutil.md

```diff
@@ -0,0 +1,32 @@
+# dockutil
+
+> Manage macOS dock items.
+> More information: <https://github.com/kcrawford/dockutil>.
+
+- Add an application to the end of the current user's dock:
+
+`dockutil --add {{path/to/application}}`
+
+- Replace one application with another in the current user's dock:
+
+`dockutil --add {{/path/to/application}} --replacing '{{dock_item_label_name}}'`
+
+- Add a directory with view options and display it as a folder icon or stack:
+
+`dockutil --add {{/path/to/directory}} --view {{grid|fan|list|auto}} --display {{folder|stack}}`
+
+- Add a URL dock item after the "Downloads" dock item:
```

Comment:
```suggestion
- Add an URL dock item after the "Downloads" dock item:
```

Reply:
"A" is correct here, because "URL" is pronounced like "yoo-ar-el", i.e. with a consonant in the first position.

Reply:
Oh, got it, sorry.